drivers/clock_control/clock_control_it8xxx2.c

```diff
@@ -0,0 +1,334 @@
+/*
+ * Copyright (c) 2022 ITE Corporation. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#define DT_DRV_COMPAT ite_it8xxx2_ecpm
+
+#include <soc.h>
+#include <zephyr/logging/log.h>
+#include "soc_espi.h"
+#include <zephyr/drivers/clock_control.h>
+#include <zephyr/dt-bindings/clock/it8xxx2_clock.h>
+#include <zephyr/drivers/clock_control/it8xxx2_clock_control.h>
+
+LOG_MODULE_REGISTER(clock_control_it8xxx2, CONFIG_CLOCK_CONTROL_LOG_LEVEL);
+
+#define PLLFREQ_MASK 0x0F
```

Comment:
Use `BIT_MASK` here.

Reply:
@fkokosinski 

Got it, I'll update both of them to #define with the BIT_MASK usage.
Truly grateful.

Reply:
Shown as follows:
`#define PLLFREQ_MASK BIT_MASK(4)
#define MCUFREQ_MASK BIT_MASK(3)`